lib/eco/incparser/astree.py

```diff
@@ -262,6 +262,18 @@ def load(self, version):
                 return
             version -= 1
 
+    def get_attr(self, attr, version):
+        if version is None:
+            return self.__getattribute__(attr)
+        version = int(version)
+        while version >= 0:
+            try:
+                return self.log[(attr, version)]
+            except KeyError:
+                version -= 1
+        raise Exception("Warning: reached zero", self, self.log)
+        return None
```

Comment:
the ``return None`` is dead code. Shouldn't this be a somewhat less general exception type?